app/viewers.py

```diff
@@ -176,8 +178,8 @@ def get_environment_summary(self, peripheral_type: str) -> Dict[str, str]:
                 reported = f'{val:.2f}'
 
             desired = "None"
-            val = environment_dict[peripheral_type]["desired"][variable]
             if variable in environment_dict[peripheral_type]["desired"]:
+                val = environment_dict[peripheral_type]["desired"][variable]
```

Comment:
Hi Jake, I think I fixed this bug (I created) in an earlier PR that @srmoore has already merged.